store/helper/helper_test.go

```diff
@@ -430,3 +432,16 @@ func mockStoreStatResponse(w http.ResponseWriter, _ *http.Request) {
 		log.Panic("write http response failed", zap.Error(err))
 	}
 }
+
+func TestComputeTiFlashStatus(t *testing.T) {
+	regionReplica := make(map[int64]int)
+	resp1 := "0\n\n"
+	resp2 := "1\n1009\n"
+	br1 := bufio.NewReader(strings.NewReader(resp1))
+	br2 := bufio.NewReader(strings.NewReader(resp2))
+	err := helper.ComputeTiFlashStatus(br1, &regionReplica)
+	require.NoError(t, err)
+	err = helper.ComputeTiFlashStatus(br2, &regionReplica)
+	require.NoError(t, err)
+	require.Equal(t, len(regionReplica), 1)
```

Comment:
How about `regionReplica` details？

Reply:
I will add this to comment

`regionReplica` is a map from RegionID to count of TiFlash Replicas in this region.

Reply:
I don't understand. The map can also check the contents.

Reply:
In previous and current implementation, we think a region A is ready if 1 TiFlash replica of it is ready. So it doesn't matter the value of map. 
But I think you are right, I should check count(content of this map), since if we have a more detailed solution, count matters.